src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -102,6 +103,15 @@ def auto_resolve_project_issues(project_id, cutoff=None, chunk_size=1000, **kwar
                 kwargs={"project_id": group.project_id, "group_id": group.id}
             )
 
+            analytics.record(
```

Comment:
@scttcper can we update the other auto-resolve to this event?

Reply:
it was deleted in https://github.com/getsentry/sentry/pull/53616